src/Components/__stories__/Search.story.tsx

```diff
@@ -1,5 +1,6 @@
+import { Box } from "@artsy/palette"
+import { storiesOf } from "@storybook/react"
```

Comment:
@pepopowitz - here's the issue 😄(And it's not totally clear...) 

```
import { storiesOf } from 'storybook/storiesOf' 
```

(https://github.com/artsy/reaction/blob/master/src/__stories__/storiesOf.js#L11)

Which will import `Boot` (and `Theme`) behind the scenes.